features/old/api/metrics/update.feature

```diff
@@ -19,10 +19,9 @@ Feature: Metric update
     And I go to the service definition page
     And I follow "Nukes deployed"
     And I fill in "Friendly name" with "Number of atomic bombs dropped"
-    And I fill in "metric[system_name]" with "bombs"
     And I fill in "Unit" with "drops"
     And I press "Update Metric"
     Then I should see "Number of atomic bombs dropped"
-    And metric "bombs" should have the following:
+    And metric "nukes" should have the following:
       | Friendly name | Number of atomic bombs dropped |
```

Comment:
Does the table stay say "Number of atomic bombs dropped" after this changes? How come? 😱

Reply:
The friendly name [was](https://github.com/3scale/porta/blob/179cd1fd95ff5ecee03618311053f2f81fb698cf/features/old/api/metrics/update.feature#L8) "Nukes deployed" and it was updated to "Number of atomic bombs dropped". The test is correct IMO.

Only thing I changed is no longer to update the `system_name`. That's why instead of searching for a metric whose `system_name` is "bombs", it now searches for "nukes", i.e. the original and unchanged `system_name`.

Reply:
aaaah riiiight sorry 🙈